modules/kinesis-firehose-honeycomb/main.tf

```diff
@@ -10,16 +10,30 @@ locals {
     local.default_lambda_parameters,
     local.user_lambda_parameters
   )
+
+  destinations = concat([{
+    honeycomb_dataset_name = var.honeycomb_dataset_name
+    honeycomb_api_key      = var.honeycomb_api_key
+    honeycomb_api_host     = var.honeycomb_api_host
+    }],
+    var.additional_destinations
+  )
+}
+
+moved {
```

Comment:
Putting `moved` here means `kinesis-firehose-honeycomb/versions.tf` should be updated to require Terraform 1.1 or greater (`required_version = ">= 1.1"`)